JSDemos/Demos/SelectBox/GroupedItems/description.md

```diff
@@ -1,6 +1,6 @@
 Data items in the SelectBox's drop-down list can be organized in groups.
 
-If the data source provides data items ungrouped, use the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/)'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to specify the data field to group by. This case is illustrated in this demo.
+If the data source provides data items ungrouped, use the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/)'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to specify the data field to group by. This case is illustrated in this demo's first and third SelectBoxes.
```

Comment:
"Provides". Save as above. You can rewrite "If the data source provides data items ungrouped" as:
1. If the data source stores ungrouped data items ...
2. If the data source does not group data items ...

Or you can come up with something better. The choice is yours.